Source/WebKitLegacy/mac/WebView/WebView.mm

```diff
@@ -1557,9 +1557,6 @@ - (void)_commonInitializationWithFrameName:(NSString *)frameName groupName:(NSSt
 #if ENABLE(NOTIFICATIONS)
     WebCore::provideNotification(_private->page, new WebNotificationClient(self));
 #endif
-#if ENABLE(DEVICE_ORIENTATION) && !PLATFORM(IOS_FAMILY)
-    WebCore::provideDeviceOrientationTo(*_private->page, *new WebDeviceOrientationClient(self));
```

Comment:
It seems like this is the only place `WebCore::provideDeviceOrientationTo()` was used. Could we also remove the implementation of this method, since it seems to be dead code now?

Reply:
Ok.